src/main/resources/static/js/location.js

```diff
@@ -14,64 +16,64 @@ function createLocation() {
 		        location: "{\"type\": \"Point\", \"coordinates\": " + loc + "}"
 		};
-		    
-		    var mydata = {
+
+			var mydata = {
 		    	entity: myloc,
 		    	string: url		
 		    };
 
 		    $.ajax({
 		            type: "POST",
 		            url: "location/create",
-		            datatype: 'json',
-		            contentType: 'application/json',
+		            datatype: "json",
+		            contentType: "application/json",
 		            data: JSON.stringify(mydata),
 		            error: function (e) {
 		                $.notify({
-		                    message: 'Location could not be created, check the Log for errors'
+		                    message: "Location could not be created, check the Log for errors"
 		                }, {
 		                    allow_dismiss: true,
-		                    type: 'danger',
+		                    type: "danger",
 		                    placement: {
 		                        from: "top",
 		                        align: "left"
 		                    },
 		                    animate: {
-		                        enter: 'animated fadeInDown',
-		                        exit: 'animated fadeOutUp'
+		                        enter: "animated fadeInDown",
+		                        exit: "animated fadeOutUp"
 		                    },
 		                    z_index: 9000
 		                });
 		                addToLog(e.responseText);
 		            },
 		            success: function (e) {
 		                $.notify({
-		                    message: 'Location created.'
+		                    message: "Location created."
 		                },{
 		                    allow_dismiss:true,
-		                    type: 'info',
+		                    type: "info",
 		                    placement: {
 		                        from: "top",
 		                        align: "left"
 		                    },
 		                    animate: {
-		                        enter: 'animated fadeInDown',
-		                        exit: 'animated fadeOutUp'
+		                        enter: "animated fadeInDown",
+		                        exit: "animated fadeOutUp"
 		                    },
 		                    z_index: 9000
 		                });
-		                addToLog('Location created.');
-		                closeModal('thingdialog');
+		                addToLog("Location created.");
+		                closeModal("thingdialog");
 
-		                var text = e.name + ' (' + e.frostId + ')';
+		                var text = e.name + " (" + e.frostId + ")";
 		                var option = new Option(text, text, null, null);
-		                option.setAttribute('data-value', JSON.stringify(e, null, 4));
+		                option.setAttribute("data-value", JSON.stringify(e, null, 4));
 
-		                $('#locations').append(option).trigger('change');
-		                $('#locations').val(text);
+		                $("#locations").append(option).trigger("change");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs. (no-mixed-spaces-and-tabs)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)